src/app/features/approver/components/approver-actions.tsx

```diff
@@ -0,0 +1,26 @@
+import { css } from 'leather-styles/css';
```

Comment:
Just a heads up, I meant to mention before, I have a `<Footer` in the containers branch we can eventually use instead of this

Reply:
Will that play nicely with the fixed positioning though?

Reply:
The footers are all supposed to be sticky now. Ignore my `Footer` for now and go as planned and then we can see if we can refactor them to share